lesson17/src/css/styles.css

```diff
@@ -0,0 +1,233 @@
+html {
+	font-size: 62.5%;
+}
+body {
+	font-family: 'Yomogi', cursive;
+	color: #666;
+	position: relative;
+	width: 100%;
+	font-size: 1.6rem;
+}
+
+h1 {
+	text-align: center;
+	margin-bottom: 30px;
+}
+
+img {
+	max-width: 100%;
+	height: auto;
+	border: none;
+	vertical-align: top;
+}
+
+a {
+	color: rgb(45, 112, 189);
+	font-size: 1.8rem;
+}
+
+.loading {
+	width: 100%;
+	height: 100vh;
+	background-color: #fff;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+  position: fixed;
+  z-index: 100;
+}
+
+.error-message{
+	background-color: #ffd6d6;
```

Comment:
Q
What is your intention in not aligning the indentation?

Reply:
Some spaces were tabs.
GitHub seems to be eight half-width characters if it is a tab space.

This has been fixed.

[a00849b](https://github.com/nyattsu72/hanson-frontend-cource/pull/25/commits/a00849bc015144b240e38f93118200de3ea8d53b)